src/routes/JobSummary/components/utils.js

```diff
@@ -22,13 +22,16 @@ export const BaseLineCell = ({ children }) => (
   </div>
 );
 
-export const getHotlinkCellFormatter = (isHotlink, entityLabel, path, entity) => {
+export const getHotlinkCellFormatter = (isHotlink, entityLabel, path, entity, stripes) => {
   if (isHotlink) {
     return (
       <BaseLineCell>
         <TextLink
           data-test-entity-name={entity}
-          to={path}
+          to={{
+            pathname: path,
+            state: { tenantTo: stripes?.okapi?.tenant }
```

Comment:
If you need `stripes` just to get the `tenant` then I suggest passing `tenant` to this unit instead of whole `stripes` obj